homework1/src/ReplaceElements.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Arrays;
+
+public class ReplaceElements {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 2, 3, 3, 10, 11, 20};
+        int numbers_length = numbers.length / 2;
+
+        for (int position = 0; position < numbers_length; position++) {
```

Comment:
okay, that's position variable in pretty readable, but in major number of cases the variable in the loop is preferable called as "i"
for the second loop it's often named "j"

Reply:
Fixed in commit: f1536a4